Public/Src/Cache/VerticalStore/MemoizationStoreAdapter/CloudStoreLocalCacheServiceFactory.cs

```diff
@@ -159,8 +159,8 @@ private sealed class Config
             [DefaultValue(60 * 60)]
             public int RocksDbMemoizationStoreGarbageCollectionIntervalInSeconds { get; set; }
 
-            [DefaultValue(60 * 60 * 2)]
-            public int RocksDbMemoizationStoreGarbageCollectionProtectionTimeInSeconds { get; set; }
+            [DefaultValue(500_000)]
+            public int RocksDbMemoizationStoreGarbageCollectionMaximumNumberOfEntries { get; set; }

```

Comment:
>RocksDbMemoizationStoreGarbageCollectionMaximumNumberOfEntries [](start = 23, length = 62)

Be sure to delete this value from the config in cbtest so we don't break things when we deploy.

Reply:
I'll make sure to disable the RocksDb in CBTest when we are ready to deploy instead. Otherwise, the GC time will force evictions before a build completes